virtualizarr/kerchunk.py

```diff
@@ -274,10 +277,11 @@ def variable_to_kerchunk_arr_refs(var: xr.Variable, var_name: str) -> KerchunkAr
                     f"Cannot serialize loaded variable {var_name}, as it is encoded with an offset"
                 )
             if "calendar" in var.encoding:
-                raise NotImplementedError(
-                    f"Cannot serialize loaded variable {var_name}, as it is encoded with a calendar"
-                )
+                print("JULIA 0 ", var)
```

Comment:
Nice to know I'm not the only person who still uses print statements as their primary method of debugging 😂

Reply:
😵‍💫  yeah I'm pretty advanced.